lib/quant/indicators/indicator_point.rb

```diff
@@ -27,6 +27,17 @@ def initialize(indicator:, tick:, source:)
       def_delegator :indicator, :dominant_cycle_kind
       def_delegator :indicator, :pivot_kind
 
+      def_delegator :tick, :high_price
+      def_delegator :tick, :low_price
+      def_delegator :tick, :close_price
+      def_delegator :tick, :open_price
+      def_delegator :tick, :volume
+      def_delegator :tick, :trades
```

Comment:
**comment**: Some of the `IndicatorPoint`'s were redeclaring high/low, etc. and these should just be delegated to the `Ticks::Tick` associated with the point.